components/ide/code/leeway.Dockerfile

```diff
@@ -46,6 +47,10 @@ RUN yarn --frozen-lockfile --network-timeout 180000
 RUN rm -rf remote/node_modules/
 COPY --from=dependencies_builder /gp-code/remote/node_modules/ /gp-code/remote/node_modules/
 
+# check that the provided codeVersion is the correct one for the given codeCommit
+RUN commitVersion=$(curl https://raw.githubusercontent.com/gitpod-io/openvscode-server/$CODE_COMMIT/package.json | jq -r .version) \
```

Comment:
We already have package.json checked out?

Reply:
True 😆, nice catch!